node-operators/networks/collator.md

```diff
@@ -174,138 +177,127 @@ console.log(`Candidate pool size is: ${candidatePool.length}`);
 
  4. 在**submit the following extrinsic**菜单中选择**parachainStaking** pallet
 
- 5. 选择**executeLeaveCandidates**函数
+ 5. 选择**executeLeaveCandidates** extrinsic
 
- 6. 选择目标候选人账户（在提交`scheduleLeaveCandidates`函数后，任何人都可以在延迟时段过后执行请求）
+ 6. 选择目标候选人账户（在提交`scheduleLeaveCandidates` extrinsic后，任何人都可以在退出生效期过后执行请求）
 
- 7. 提交交易。根据向导指引使用创建账户时的密码进行交易签名
+  7. 提交交易。根据向导指引使用创建账户时的密码进行交易签名
 
 ![Execute leave candidates request](/images/node-operators/networks/collators/collator-polkadotjs-10.png)
 
-#### 取消离开候选人的请求 {: #cancel-request-to-leave-candidates }
+### 取消离开候选人的请求
```

Comment:
this should have this attribute still `{: #cancel-request-to-leave-candidates }`